controllers/controller.go

```diff
@@ -75,7 +75,7 @@ func FromConfiguration(fullConfig config.Configuration, spConfig config.ServiceP
 		return nil, err
 	}
 
-	ts, err := tokenstorage.New(cl)
+	ts, err := tokenstorage.NewVaultStorage(cl, "spi-oauth", fullConfig.VaultHost, fullConfig.ServiceAccountTokenFilePath)
```

Comment:
Shouldn't this be the notifying token storage?

Reply:
it is, it's inside the function https://github.com/redhat-appstudio/service-provider-integration-operator/pull/88/files#diff-9690267b7b50c62307b865c258b7a407dca24db90457879bc6c43a266a9546cfR61

Reply:
good point. yes it is used. I'll fix it

Reply:
fixed